### DISCUSSION TITLE
How to disable vim mode?

### DISCUSSION THREAD

**Discussion by qx-775** [starship/starship#3080] [Q&A] [answered]
I was redirected here to ask questions.

So I believe starship prompt allows for vim-mode with insert/select mode although it is not documented enough IMO.
Could not find any github issues on how to disable this too..
I absolutely hate vim mode in the terminal, and I want to disable it on starship. How do i achieve this?

Also I wish to quickly go the the next word on the prompt by pressing control+arrow key to quickly move from word to word in the prompt. I believe oh-my-zsh enables this by default. Does anyone know how I can achieve this in starship?

I am using zinit with starship prompt but I think my step 1 mistake is I don't know where vi-mode is enabled and how I can disable it.
Is it coming from ZSH itself, zinit or starship? I believe it to be the latter.

**Comment by davidkna** [comment, marked as answer]
Starship does not enable or disable vi-mode and I don't think zinit, zsh itself or oh-my-zsh enable it by default either.

You can switch back to emacs mode with `set -o emacs`.

**Comment by qx-775** [comment]
Alright, that is helpful. My quest to find out what is causing vi-mode to be enabled and learning more about ZSH will continue.
Thanks for your help.

**Comment by David-Kyrat** [comment]
I also have the same problem. Since the starship prompted reflected changes on whether i was in vim mode or normal mode in the terminal, i thought that it was responsible for it.
(I use wezterm + zsh (without oh-my-zsh or any other thing) + starship)
